adafruit_rfm69.py

```diff
@@ -474,8 +474,10 @@ def operation_mode(self, val):
         op_mode |= val << 2
         self._write_u8(_REG_OP_MODE, op_mode)
         # Wait for mode to change by polling interrupt bit.
+        start = time.monotonic()
         while not self.mode_ready:
-            pass
+            if (time.monotonic() - start) >= 1:
+                raise RuntimeError('Operation mode failed to set.')
```

Comment:
black wants double quotes here...

Reply:
I'll change to double quotes. Thanks for the link about Black.